### PR TITLE
Corrected False EMS Sensor Error

### PR DESCRIPTION
**Pull request by rglazer2**
Addresses issue #4941

### CODE REVIEW

**Comment by mjwitte** [comment]
@rglazer2 Interesting that this results in a change in the sorted order of output variable names in _5ZoneEvapCooled.idf which apparently is the only test file that uses the sort by Name option for the RDD/MDD output:
  Output:VariableDictionary,IDF,Name;
Are there any other uses of SortAndSetup that we need to review?

File: tst/EnergyPlus/unit/SortAndStringUtilities.unit.cc [hunk 1]
```diff
@@ -23,3 +25,31 @@ TEST( SortAndStringUtilitiesTest, Basic )
...
+	int NumObjectDefs = ListOfObjects.size();
+
+	iListOfObjects.allocate(NumObjectDefs);
+	SetupAndSort(ListOfObjects, iListOfObjects);		//list is resorted
+
+	auto index = FindItemInSortedList("SYSTEM NODE TEMPERATURE", ListOfObjects, NumObjectDefs);
+
+	EXPECT_EQ(3, index);
+
+}
```
Comment (mjwitte) [inline tst/EnergyPlus/unit/SortAndStringUtilities.unit.cc]: @rglazer2 Need a newline here.

**Comment by mjwitte** [comment]
@rglazer2 Looking again at the diffs on _5ZoneEvapCooled.idf, this is an improvement because the sort is case insensitive now.  So everything is alphabetical now regardless of case, where before, for example, "BSDF" sorted before "Back".

**Comment by rglazer2** [comment]
@mjwitte Anything else that needs to be done for this?

**Comment by mjwitte** [comment]
@rglazer2 Not that I see.  Ready for @Myoldmopar to review.

**Comment by Myoldmopar** [comment]
@rglazer2 @mjwitte I like these changes.  If CI comes back happy I'll merge it.

**Pull request merged**